doc/ref_arch/openstack/chapters/chapter08.md

```diff
@@ -19,6 +19,8 @@ The most obvious way to address the gaps will be to propose a set of APIs in the
 
 * Public and Multi-Cloud  - The framework offered by OpenStack does not support multi-cloud deployments, along with the core deployments
```

Comment:
@sukhdevkapur and @walterkozlowski 

Do we first need to define the problem more clearly and as to what is meant by and needed for multi-cloud deployments? The needs maybe in terms of use cases or feature functionality.

Then, do we need to define a set of requirements in RM or in RA's for multi-cloud deployments?

Maybe then, in the RAs, we can specify how are we going to satisfy these requirements.

Critique?